src/domotica/domotica_cv.c

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file domotica_cv.c
+ * @brief Process CV numbers for the loconet domotica module
+ *
+ * \copyright Copyright 2017 /Dev. All rights reserved.
+ * \license This project is released under MIT license.
+ *
+ * @author Jan Martijn van der Werf <janmartijn@slashdev.nl>
+ */
+
+#include "domotica_cv.h"
+
+// ----------------------------------------------------------------------------
+void loconet_cv_written_event(uint16_t lncv_number, uint16_t value)
+{
+  if (lncv_number == 0)
+  {
+    loconet_config.bit.ADDRESS = value;
+  }
+  else if (lncv_number == 2)
+  {
+    loconet_config.bit.PRIORITY = value;
+  }
+  else if (lncv_number == 3)
+  {
+    // Fast clock setting
+  }
+  else if (lncv_number >= DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_START && lncv_number < DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_END)
+  {
+    // Output light settings
+    if (value <= 0xFF)
+    {
+      domotica_set_output_brightness(lncv_number - DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_START, (uint8_t) value);
+    }
+  }
+  else if (lncv_number >= DOMOTICA_LNCV_INPUT_ADDRESSES_START && lncv_number < DOMOTICA_LNCV_INPUT_ADDRESSES_END)
```

Comment:
True. I've updated the if statements, as those are implied (I added this as comments between the "else-if" clauses. For the macros, I left them in, because of readability of the definitions.